File: C/1010.c
#include <stdio.h>

float main(){
	int codigo1, numero1, codigo2, numero2;
	float valor_unitario1, valor_unitario2, total, valortotal1, valortotal2;
	scanf ("%d%d%f", &codigo1, &numero1, &valor_unitario1);
	scanf ("%d%d%f", &codigo2, &numero2, &valor_unitario2);
	
					//PRIMEIRA//
	valortotal1 = numero1 * valor_unitario1;
	
					//SEGUNDA// 
	valortotal2 = numero2 * valor_unitario2;
	
	
	total = valortotal1 + valortotal2;
	printf("VALOR A PAGAR: R$ %.2f\n", total);
	
	return 0;
}

File: Python/1010.py
from decimal import Decimal 
codigo1, numero1, valor1 = input().split(" ")
codigo1 = int(codigo1)
numero1 = int(numero1)
valor1 = float(valor1)
codigo2, numero2, valor2 = input().split(" ")
codigo2 = int(codigo2)
numero2 = int(numero2)
valor2 = float(valor2)
total = ((numero1*valor1)+(numero2*valor2))
total = Decimal("%.2f"%total)
print("VALOR A PAGAR: R$ "+str(total))